modules/network-observability-loki-install.adoc

```diff
@@ -33,8 +32,8 @@ There are several ways you can install Loki. One way is by installing the Loki O
 .. Verify that *Loki Operator* is listed with *Status* as *Succeeded* in all the projects.
 +
 . Create a `Secret` YAML file. You can create this secret in the web console or CLI. 
-
-.. Using the web console, navigate to the Import icon ,*+*, in the top right corner. Drop your YAML file into the editor. It is important to create this YAML file in the `netobserv` namespace that uses the `access_key_id` and `access_key_secret` to specify your credentials. 
+.. Using the web console, navigate to the *Project: All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.
```

Comment:
I haven't seen the `*Projects: All Project*` part be formatted this way in other parts of the docs when referencing the web console. I think even in this context it would be:
`*Projects* -> * All Projects*`

Reply:
You're right- this is a mistake on my part.